### PR TITLE
Improve PV creation logging

### PR DESCRIPTION
**Pull request by Madhu-1**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug

> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
log PV object with v(5) verbosity.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>
**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes-csi/external-provisioner/issues/329

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -617,8 +617,8 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 		}
 	}
 
-	klog.Infof("successfully created PV %+v", pv.Spec.PersistentVolumeSource)
-
+	klog.Infof("successfully created PV %v for PVC %v driver name %v csi volume name %v", pv.Name, options.PVC.Name, pv.Spec.CSI.Driver, pv.Spec.CSI.VolumeHandle)
```
Comment (msau42) [inline pkg/controller/controller.go]: Explicitly set a log level here.  How about 2?

Comment (msau42) [reply pkg/controller/controller.go]: I don't think we need to log driver name here? The external-provisioner only handles one driver.

Comment (Madhu-1) [reply pkg/controller/controller.go]: yes you  are right, removed driver name logging and set the log level to 2

**Pull request merged**